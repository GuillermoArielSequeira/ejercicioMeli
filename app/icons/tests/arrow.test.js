const React = require('react');
const { shallow } = require('enzyme');
const Arrow = require('../arrow');

describe('Arrow svg Component', () => {
  it('render Arrow correctly', () => {
    const wrapper = shallow(<Arrow />);
    expect(wrapper).toMatchSnapshot();
  })
});
