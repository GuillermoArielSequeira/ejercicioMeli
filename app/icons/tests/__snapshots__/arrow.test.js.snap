// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow svg Component render Arrow correctly 1`] = `ShallowWrapper {}`;
